packages/template-retail-react-app/app/components/forms/address-fields.jsx

```diff
@@ -23,11 +23,15 @@ const defaultFormTitleAriaLabel = defineMessage({
     id: 'use_address_fields.label.address_form'
 })
 
-const AddressFields = ({form, prefix = '', formTitleAriaLabel = defaultFormTitleAriaLabel}) => {
+const AddressFields = ({
+    form,
+    prefix = '',
+    formTitleAriaLabel = defaultFormTitleAriaLabel,
+    isBillingAddress = false
```

Comment:
Is there a test covering `isBillingAddress = false` or `isBillingAddress = true` - may be in the cart tests?  if not can we add one?

Reply:
isBillingAddress = false is the shipping section and `isBillingAddress = true` is the billing address section. I've modified the checkout page to edit the billing section. I find it should give enough cover for the change in this PR. Let me know if you think we still need something else. Thanks